src/bci_build/package.py

```diff
@@ -1691,6 +1691,30 @@ def init_container_kwargs(os_version: OsVersion):
     for os_version in (OsVersion.SP4, OsVersion.TUMBLEWEED)
 ]
 
+BLACKBOX_EXPORTER_PACKAGE_NAME = "prometheus-blackbox_exporter"
+BLACKBOX_EXPORTER_CONTAINERS = [
+    ApplicationStackContainer(
+        package_name="blackbox_exporter-image",
+        os_version=os_version,
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        name="blackbox_exporter",
+        pretty_name="Blackbox exporter",
```

Comment:
nitpick, this will make the description look more consistent (everything is then capitalized)
```suggestion
        pretty_name="Blackbox Exporter",
```